Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -522,3 +522,25 @@ lemma LinearEquiv.extendScalarsOfSurjective_symm (f : M ≃ₗ[R] N) :
     (f.extendScalarsOfSurjective h).symm = f.symm.extendScalarsOfSurjective h := rfl
 
 end surjective
+
+namespace algebraMap
+section CommSemiringCommSemiring
+
+variable {R A : Type*} [CommSemiring R] [CommSemiring A] [Algebra R A]
+   {ι : Type*} {s : Finset ι} (a : ι → R)
```

Comment:
As I understand it, the preference is to avoid implicitly bound variables. Can you move the `(a : ι → R)` into the statement, please? (Or a maintainer say this is fine.)

Reply:
I agree that `a` should be an explicit argument, not a `variable`.